docs/how-to/analytics-dashboard.md

```diff
@@ -6,49 +6,46 @@ slug: /analytics-dashboard
+Amplication's Analytics Dashboard is a powerful tool that provides developers and managers with valuable insights into their application's development process.
 
-The dashboard highlights key metrics related to productivity, efficiency, and code quality to help you understand the impact of using Amplication on your development workflow and the overall return on investment.
+The dashboard highlights key metrics related to productivity, efficiency, and code quality to help you understand the impact of using Amplication on your development workflow.
 
 To start leveraging the insights provided by the Analytics Dashboard, navigate to the Workspace Overview or Project Overview screen in your Amplication account.
 
 ## Productivity Metrics
 
-The Productivity Metrics section provides an overview of the development activity and code generation. This lets developers track their progress and identify trends in their development process using the **Overtime Graph** and **Generated Lines Of Code** sections.
+The Productivity Metrics section provides an overview of the development activity and code generation. This lets developers and managers track their progress using the **Overtime Graph**, **Generated Lines Of Code**, and **Time Saved** sections.
 
 ### Overtime Graph
 
-The Overtime Graph is a visual representation of the development activity over a specified period. It displays the number of builds, entities, APIs, and plugins created or updated, allowing developers to track their progress and identify trends in their development process.
+The Overtime Graph is a visual representation of the development activity over a specified period. It displays the number of builds generated and the number of entities, APIs, and plugins created or updated during the selected time frame.
 
-Developers can customize the time range to view data from the last month, last 3 months, last year, or a custom date range (not less than 1 month). The default view shows data from the last year, displayed in months.
+Developers can customize the time range to view data from the last month, last 3 months, or last year. The default view shows data from the last year, displayed in months.
 
-The Overtime Graph is interactive, allowing developers to select or deselect specific data points (builds, entities, APIs, or plugins) to focus on the metrics that matter most to them.
+### Time Saved
+
+The Time Saved metric estimates the amount of time saved by using Amplication-generated code for your Node.js backend. Based on industry standards and considering the efficiency of Node.js backend developers, we estimate that a developer creates around 8,335 lines of code per month. This breaks down to approximately 52.05 lines of code per developer hour.
+
+By quantifying the time saved, developers can better understand the impact of Amplication on their development velocity and make informed decisions about resource allocation and project planning.
 
 ### Generated Lines of Code (LOC)
```

Comment:
I think the LOC explanation should appear before the time saved (as the time saved "depends" on the LOC metric)